hydra-node/src/Hydra/API/RestServer.hs

```diff
@@ -1,13 +1,36 @@
+{-# LANGUAGE PatternSynonyms #-}
 {-# LANGUAGE UndecidableInstances #-}
 
 module Hydra.API.RestServer where
 
 import Hydra.Prelude
 
+import qualified Cardano.Api.UTxO as UTxO
 import Cardano.Binary (decodeFull', serialize')
-import Data.Aeson (Value (String), object, withObject, (.:), (.=))
+import Data.Aeson (Value (Object, String), object, withObject, (.:), (.=))
+import qualified Data.Aeson as Aeson
+import qualified Data.Aeson.KeyMap as KeyMap
 import qualified Data.ByteString.Base16 as Base16
-import Hydra.Ledger (IsTx, UTxOType)
+import Data.ByteString.Short ()
+import Hydra.Cardano.Api (
+  BuildTx,
+  BuildTxWith (..),
+  CtxUTxO,
+  HashableScriptData,
+  PlutusScript,
+  ScriptDatum (ScriptDatumForTxIn),
+  ScriptWitnessInCtx (ScriptWitnessForSpending),
+  TxIn,
+  TxOut,
+  UTxO',
+  WitCtxTxIn,
+  Witness,
+  mkScriptWitness,
+  pattern ScriptWitness,
+ )
+import Hydra.Cardano.Api.Prelude (Era, ScriptWitness)
+import Hydra.Ledger (IsTx)
+import Hydra.Ledger.Cardano ()
 
 newtype DraftCommitTxResponse tx = DraftCommitTxResponse
```

Comment:
This would even address #945 if we define

```hs
toJSON (DraftCommitTxResponse tx) =
    object
      [ "commitTx" .= (String . decodeUtf8 . Base16.encode $ serialiseToCBOR tx)
      ]
```

and `FromJSON` using `deserialiseFromCBOR`.

But then, the other `ToJSON` occurrences where instances of `Tx` are used are still broken/different. CC @v0d1ch